src/test/regress/Makefile

```diff
@@ -10,6 +10,10 @@ ifndef MAJORVERSION
 MAJORVERSION := $(basename $(VERSION))
 endif
 
+ifeq ($(TESTHLL),true)
+HLLEXT := "--load-extension=hll"
```

Comment:
Good call. I forget not all extensions actually need to be loaded. I'll switch to that.